server/api/initial_data.py

```diff
@@ -894,6 +903,34 @@ def _create_project_summaries(db, db_session):
     db._upsert(db_session, project_summaries, ignore=True)
 
 
+def _align_schedule_labels(db, db_session):
+    db_schedules: list[mlrun.common.schemas.ScheduleRecord] = db.list_schedules(
+        session=db_session
+    )
+    schedules_update = []
+    for db_schedule in db_schedules:
```

Comment:
I'm thinking that we need to optimize this for scale systems. I would take the following steps:
1. Add `as_records` flag similar to `list_artifacts`.
2. Use sqlalchemy lazy load - return the query and then for loop on the query instead of using query.all() and loading all schedules in memory.
3. Transform the DB schedule to schema schedule then you have both and you don't need to get the DB one (saves us O(n) queries to the DB/cache).
```suggestion
    schedules_update = []
    for db_schedule in db.list_schedules(session=db_session, as_records=True):
        schedule_record = db._transform_schedule_record_to_scheme(db_schedule)
```